test/unit/struct_tree_spec.js

```diff
@@ -0,0 +1,107 @@
+/* Copyright 2017 Mozilla Foundation
```

Comment:
While the current year isn't really a great one so far, we probably want to use it here anyway :-)